cloud/endagaweb/templates/dashboard/report/call-sms.html

```diff
@@ -0,0 +1,154 @@
+{% extends "dashboard/layout.html" %}
+{% comment %}
+Copyright (c) 2016-present, Facebook, Inc.
+All rights reserved.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree. An additional grant
+of patent rights can be found in the PATENTS file in the same directory.
+{% endcomment %}
+{% load apptags %}
+{% load humanize %}
+{% load crispy_forms_tags %}
+
+
+{% block title %}
+    {% tmpl_const "SITENAME" %} | Report
+    {% if report_summary  %}
+        | "{{ report_summary }}"
+    {% endif %}
+{% endblock %}
+
+{% block pagestyle %}
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/nvd3/1.1.13-beta/nv.d3.min.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datetimepicker/4.14.30/css/bootstrap-datetimepicker.min.css">
+<link rel="stylesheet" href="https://cdn.datatables.net/1.10.15/css/jquery.dataTables.min.css" type="text/css"/>
+{% endblock %}
+
+{% block content %}
+    {% include "dashboard/report/header.html" with header='Call & SMS' %}
+
+    <div class='row'>
+        {% include "dashboard/report/nav.html" with active_tab='call_sms' %}
+
+        <div class='content col-xs-12 col-sm-10 col-md-10'>
+
+            {% include "dashboard/report/filter.html" with action_url='/dashboard/reports/calls' %}
+            {% if network_has_activity %}
+            <div class="row">
+              <div id='sms-usage-chart'></div>
+            </div>
+            <div class="row">
+              <div id='call-number-chart'></div>
+            </div>
+            <div class="row">
+              <div id='call-minutes-chart'></div>
+            </div>
+            
+            <div class="row">
+                {% include 'dashboard/timezone-notice.html' %}
+            </div>
+            {% else %}
+                <p>There is no network activity to display.</p>
+            {% endif %}
+        </div> <!-- /.col-md-4 -->
+    </div>
+{% endblock %}
+
+
+{% block js %}
+{% if network_has_activity %}
+<script>
+  $(function() {
+    setTimeout(function() {
+      $('.timezone-notice').fadeIn(500);
+    }, 800);
+  });
+  $("#save-btn").click(function() {
+      $("#call-number-chart").get(0).toBlob(function(blob) {
+        saveAs(blob, "call-number-chart.png");
+    });
+});
+</script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/react/0.12.2/react.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/react/0.12.2/JSXTransformer.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.1.5/d3.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/nvd3/1.1.13-beta/nv.d3.min.js"></script>
```

Comment:
This is for chat js files. Above includes files are css style. CSS files are included in top of page in 'pagestyle' block to load pages fast, and js files are added in bottom '"js" block